.vscode/settings.json

```diff
@@ -12,6 +12,7 @@
   "eslint.enable": true,
   "eslint.format.enable": true,
   "eslint.lintTask.enable": true,
+  "eslint.experimental.useFlatConfig": true,
   "eslint.options": {
     "overrideConfigFile": "./package.json",
```

Comment:
idk if this option is relevant anymore

Reply:
why remove ln 21?

Reply:
deprecated now: 
![image](https://github.com/iTwin/itwinjs-core/assets/98850418/a5e92956-07c1-4b4a-ac1e-3480e43c9bdd)

Additionally, I pulled in this branch and tested on VS Code locally and ESLint was showing errors correctly with those lines removed!

Reply:
Thanks, will try it out